lib/tessel/provision.js

```diff
@@ -188,9 +223,5 @@ function AlreadyAuthenticatedError() {
 util.inherits(AlreadyAuthenticatedError, Error);
 
 actions.AlreadyAuthenticatedError = AlreadyAuthenticatedError;
-
 module.exports = actions;
-
-if (global.IS_TEST_ENV) {
-  module.exports = actions;
-}
+module.exports.TESSEL_AUTH_KEY = authKey;
```

Comment:
This isn't necessary, because the key is already exposed on `Tessel.TESSEL_AUTH_KEY`. There should be one source of truth here, and the accessor is much safer